.pkgr.yml

```diff
@@ -0,0 +1,2 @@
+targets:
```

Comment:
Would it be possible to put this file in a top-level config/ directory within the project?

Reply:
Packager requires it to be here or it won't pick it up.

Reply:
I think I'm okay with this being at the top level because it's a ".-" hidden file.

Reply:
Just double checked, the .pkgr.yml is the only supported way I can find: http://doc.packager.io/documentation/customizing-the-build/